main.tf

```diff
@@ -700,7 +700,7 @@ module "aws_for_fluentbit" {
     },
     {
       name  = "cloudWatchLogs.logGroupName"
-      value = local.aws_for_fluentbit_cw_log_group_name
+      value = local.aws_for_fluentbit_cw_log_group_name != null ? local.aws_for_fluentbit_cw_log_group_name : ""
```

Comment:
isn't this already covered on line 570? should we just change the fall back condition on line 570 from `null` to `""`?

Reply:
thank you for your reply.
I changed line 703, thinking that the range of influence would be smaller.
However, when `aws_for_fluentbit_cw_log_group.create = false`, only line 703 references `aws_for_fluentbit_cw_log_group_name`.

There is no need to change line 703, so I changed line 570 to "". https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/pull/271/commits/681b87c90fa9197775ace915252b4d6d073c316a